src/styles/print.css

```diff
@@ -27,6 +27,7 @@
   .h5p-course-presentation .h5p-slides-wrapper {
     height: 100%;
     background: transparent;
+    margin-bottom: 100%;
```

Comment:
you can't print paddings and margins so it works to simulate a full height if heights properties do not work :C